software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -388,20 +401,38 @@ public static String getDefaultUserMetadataString() {
                 "}"
         ));
 
+        // FIXME USE_SECURE_WINRM
+        // Missing generate certificate step.
+        // https://support.microsoft.com/en-us/kb/2019527
+        //
+        // One possible approach is to generate a certificate and append it to this command.
+        // http://stackoverflow.com/questions/1615871/creating-an-x509-certificate-in-java-without-bouncycastle
+        boolean useSecureWinrm = config.getBag().get(USE_SECURE_WINRM);
+
+        boolean credSsp = config.getBag().get(USE_NTLM);
```

Comment:
SSP is not NTLM. Better leave it alone (or use separate config key).
`Negotiate` includes both `Kerberos` and `NTLM` (but not sure what happens when `Kerberos` is explicitly disabled).